packages/react-slider/src/components/Slider/index.ts

```diff
@@ -1,5 +1,6 @@
 export * from './Slider';
-export * from './Slider.types';
+// Explicit exports to ommit SliderCommons
+export type { SliderProps, SliderSlots, SliderState, SliderOnChangeData } from './Slider.types';
```

Comment:
nit: Order these alphabetically
```suggestion
export type { SliderOnChangeData, SliderProps, SliderSlots, SliderState } from './Slider.types';
```